main.js

```diff
@@ -1,52 +1,56 @@
 
 let dotsCount = 0,
 
-singleDots = ["ب", "ج", "خ", "ذ", "ز", "ض", "ظ", "غ", "ف", "ن"],
-doubleDots = ["ت", "ق"],
-tripleDots = ["پ", "ث", "چ", "ژ", "ش"],
-midLetterDots = ["ی"];
+singleDots = ["ب", "ج", "خ", "ذ", "ز", "ض", "ظ", "غ", "ف", "ن"]
+doubleDots = ["ت", "ق"]
+tripleDots = ["پ", "ث", "چ", "ژ", "ش"]
+midLetterDots = ["ی"]
 
 
-inputTag = document.getElementById("text"),
-result = document.getElementById("dots");
-
+inputTag = document.getElementById("text")
+result = document.getElementById("dots")
 
+/**
+ * counts dots
+ * @returns {void}
+ * @param text
+ */
 function dotCounter(text) {
 
 
-    var spText = text.split("");
+    var spText = text.split("")
 
     for (let i = 0; i < spText.length; i++) {
-        const letter = spText[i];
+        const letter = spText[i]
 
  
         if (singleDots.includes(letter)) {
-            dotsCount++;
+            dotsCount++
         };
         if (doubleDots.includes(letter)) {
-            dotsCount+= 2;
+            dotsCount+= 2
         };
         if (tripleDots.includes(letter)) {
-            dotsCount+= 3;
+            dotsCount+= 3
         }
 
 
         if (midLetterDots.includes(letter) && i !== spText.length - 1) {
             if (spText[i+1] !== " ") {
-                dotsCount+= 2;
+                dotsCount+= 2
             }
         }
     }
-};
+}
 
 inputTag.addEventListener("input", function () {
 
 
-    dotsCount = 0;
+    dotsCount = 0
 
 
-    dotCounter(inputTag.value);
+    dotCounter(inputTag.value)
```

Comment:
what is your problem with semocolons dude?😂😔

Reply:
If u want to use them, you are supposed to use them all over the place, not one here maybe one there
and "commit" was not about semicolons and extra commas anyway!
After all it is an educational project